test-cases/features/full-cluster-replacement-multidc.yaml

```diff
@@ -0,0 +1,22 @@
+test_duration: 360
+
+prepare_write_cmd:  ["cassandra-stress write cl=QUORUM n=5048570 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=80 -pop seq=1..5048570 -col 'n=FIXED(8) size=FIXED(128)' -log interval=5",
```

Comment:
The task talks about a much bigger data set, 5gb is kind of trivial. and is enough to just test the code flow

Reply:
Ok so how much we want for this?
I think we need specific definitions for all the test cases in dtest/SCT/Nemesis in the [test plan](https://docs.google.com/document/d/1aKCTs6SiH9ORDYvkAQ8upOpVwZqaxsW9VfbJqKG63g0/edit) and [test cases](https://docs.google.com/spreadsheets/d/1nWNU50dKVz12use1VSClwonbcCwQPmnuk0OIPg589ec/edit#gid=0) docs.
The docs are still in draft state and need to be improved.

Reply:
The plan didn't define it, but from my POV

There are two things to consider

1) you want a basic case like the individual nemesis to make sure flow is o.k. (again coming back to why nemesis kind of fit this better), and for that you don't need a very big data set

2) achieving the general goal of the plan, it is properly gonna be 6-9 nodes (multi DC, multi az), and a big data set, how big, I don't know it depends on the context of that task/plan, which I don't have.

We have tests with data sets from 100gb to 1tb

5gb is just a starting position we set for validation of nemesis

Maybe @roydahan has more context about the goal of this goal